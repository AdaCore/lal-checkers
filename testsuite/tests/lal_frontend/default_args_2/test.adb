procedure Test is
   procedure F(R : out Integer; X : Integer := 0) is
   begin
      null;
   end F;

   R : Integer;
begin
   F(R);
end Ex1;
