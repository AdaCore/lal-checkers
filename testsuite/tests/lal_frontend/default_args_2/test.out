Program:
  R#1 = Input()
  ret0#2 = <SubpBody ["F"] test.adb:2:4-5:10>(R#1, 0)
  R#1 = Get_0(ret0#2)
  end:
